CHANGELOG.latest.md

```diff
@@ -1,2 +1,14 @@
+### New Features
+* Add `PaymentMode` on to `PricingPhase` to describe behavior for offers (#943) via Josh Holtz (@joshdholtz)
+* Compute CustomerInfo from a list of purchases (#885) via Cesar de la Vega (@vegaro)
```

Comment:
I guess this could be moved to `Other changes`... It's not offering a new feature to developers just yet.

Reply:
Ah yup! Just made this change and pushed it 😊  Caught it after I created PR 🤦‍♂️